examples/features/scale-from-zero/autoscale-netsvc.yaml

```diff
@@ -9,9 +9,12 @@ spec:
   name: autoscale-icmp-responder
```

Comment:
We also fixed name duplication https://github.com/networkservicemesh/deployments-k8s/issues/2008 in sdk-k8s. Could you also remove it? :) 
```suggestion
```

Reply:
ok, done